oio/container_streaming/app.py

```diff
@@ -472,7 +471,7 @@ def _do_get(self, req, account, container):
         if 'Range' not in req.headers:
             # TODO: instead expanding blocks to a full list,
             # use [first_block - last_block]
-            tar = TarStreaming(self.conn, account, container, range(blocks),
+            tar = TarStreaming(self.proxy, account, container, range(blocks),
```

Comment:
The word "streaming" is bothering me. This class is not a "streaming". It may be a "stream". In the docstring you say this class acts as a "File Object", and each instance is dedicated to a single container, why not `ContainerTarFile` or `ContainerTarStream`?